packages/modules/src/utils/Types.ts

```diff
@@ -31,17 +32,12 @@ export interface BatchedSessionRouterModuleConfig extends BaseValidationModuleCo
   sessionManagerModuleAddress?: string;
   nodeClientUrl?: string;
   smartAccountAddress: string;
-  storageType?: StorageType;
```

Comment:
can we not remove this storageType please

Reply:
Fixed